src/main/java/io/spring/batch/BatchGemfireFileSortApplication.java

```diff
@@ -5,16 +5,30 @@
 import java.util.List;
 import java.util.Properties;
 
+import org.apache.geode.cache.Region;
 import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.task.configuration.EnableTask;
+import org.springframework.context.ApplicationContext;
+import org.springframework.data.gemfire.config.annotation.EnableEntityDefinedRegions;
+import org.springframework.data.gemfire.config.annotation.PeerCacheApplication;
+import org.springframework.data.gemfire.mapping.MappingPdxSerializer;
+
+import javax.annotation.PostConstruct;
 
 @EnableTask
 @EnableBatchProcessing
 @SpringBootApplication
+@PeerCacheApplication
```

Comment:
By moving this here, does the master become one of the cache nodes?  We don't want that if possible (I think)...

Reply:
Yes. We may have to give it a dummy partition or Investigate this further.  I suspect it's a lifecycle/ordering issue